app/models/collection.rb

```diff
@@ -168,6 +168,9 @@ def title
   scope :for_blurb, -> { includes(:parent, :moderators, :children, :collection_preference, owners: [:user]).with_attached_icon }
 
   def cleanup_url
+    plausible_url_regex = %r{\A(?:https?://|www\.|\S+\.\S+)}i
+    return if header_image_url !~ plausible_url_regex
+    
```

Comment:
We'd like to deal with this by rescuing the exception instead of running another regex, e.g. how it was done in #5298.